lib/mix/tasks/compile.make.ex

```diff
@@ -83,6 +83,11 @@ defmodule Mix.Tasks.Compile.ElixirMake do
   and `mix test` commands.
   """
 
+  @exec_notfound_msg """
```

Comment:
We use this pattern for OS-specific messages already so I think it's good to have this here :)